tests/snappi_tests/multidut/ecn/test_multidut_dequeue_ecn_with_snappi.py

```diff
@@ -107,12 +113,26 @@ def test_dequeue_ecn(request,
                            snappi_extra_params=snappi_extra_params)[0]
 
     logger.info("Running verification for ECN dequeue test")
+
     # Check if all the packets are captured
     pytest_assert(len(ip_pkts) == data_flow_pkt_count,
                   'Only capture {}/{} IP packets'.format(len(ip_pkts), data_flow_pkt_count))
 
-    # Check if the first packet is ECN marked
-    pytest_assert(is_ecn_marked(ip_pkts[0]), "The first packet should be marked")
+    if (("platform_asic" in duthost1.facts) and (duthost1.facts["platform_asic"] == "broadcom-dnx")):
```

Comment:
@amitpawar12 please provide more detail on this logic. which range of packets are not marked and which ones marked.